pages/windows/get-childitem.md

```diff
@@ -0,0 +1,24 @@
+# Get-ChildItem
+
+> List items in a directory.
+> More information: <https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-childitem>
```

Comment:
We don't use the i18n in Microsoft URLs
```suggestion
> More information: <https://docs.microsoft.com/powershell/module/microsoft.powershell.management/get-childitem>.
```

Reply:
I see, I wasn't aware of that. Thank you!